style.css

```diff
@@ -130,3 +130,465 @@ p {
 .menu-bar {
   display: none;
 }
+
+/* work section */
+.work {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: 100%;
+}
+
+.work-1 {
+  display: flex;
+  flex-direction: column;
+}
+
+.work1-visual {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.work1-visual > h1 {
+  font-family: "Crete Round", serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0;
+  align-items: center;
+  color: #1c1a19;
+  margin-left: 10px;
+  margin-top: 80px;
+}
+
+.line {
+  margin-top: -90px;
+  width: 327px;
+  height: 0;
+  border: 1px solid #212121;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.work1-content {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.work-placeholder > img {
+  width: 100%;
+  margin-top: 80px;
+}
+
+.content-data {
+  width: 327px;
+  margin: auto;
+}
+
+.content-data > h1 {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.content-data > p {
+  width: 327px;
+  height: 147px;
+  margin-top: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.content-data > ul {
+  display: flex;
+  justify-content: space-between;
+  gap: 12px;
+  list-style: none;
+  padding-inline-start: inherit;
+}
+
+.list2 {
+  border: 1px solid white;
+  padding: 12px;
+}
+
+.list1 > a {
+  text-decoration: none;
+  padding: 12px;
+  text-align-last: center;
+  border: 1px solid black;
+  height: 40px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #091e42;
+}
+
+.work1-btn {
+  list-style: none;
+}
+
+.work1-button {
+  padding: 12px;
+  width: 161px;
+  background-color: rgb(255, 107, 0);
+  border: none;
+  color: white;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 10px;
+}
+
+.work1-btn :hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.work-display {
+  display: grid;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.work-2 {
+  width: 327px;
+  margin: auto;
+  margin-top: 24px;
+  background: url(./svg/ImgPlaceholder1.svg);
+  background-color: #1c1a19;
+  background-repeat: no-repeat;
+  background-position-x: center;
+}
```

Comment:
- [ ]  Great effort on your part! However, it appears that you have not met the requirements outlined by the previous reviewer. I encourage you to carefully review their feedback and make the necessary adjustments. Take the time to understand the design requirements and ensure that your implementation aligns with them. By addressing the reviewer's requests, you will enhance the overall quality of your work. Keep up the good work and strive to meet the given requirements. 

![image](https://github.com/sulaimanahmadi/Portfolio/assets/55840999/b9065c68-ace0-4eb6-88e2-b38690c0955d)

Reply:
- [ ] Kindly follow the instruction from the previous reviewers on using <code>flexbox</code> to align your content. It would make your webpage responsive 😊